src/style.css

```
@@ -0,0 +1,82 @@
+.mainBlock {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.block {
+  height: 300px;
+  min-width: 300px;
+  font-family: Arial, Helvetica, sans-serif;
+  color: white;
+  font-size: 100px;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  align-items: center;
+  width: 30%;
+}
+
+.one {
+  background-color: rgb(255,0,0);
+  order: 1;
+  flex-grow: 1;
+}
+
+.two {
+  background-color: rgb(204,0,0);
+  order: 2;
+  flex-grow: 1;
+}
+
+.three {
+  background-color: rgb(153,0,0);
+  order: 3;
+  flex-grow: 1;
+}
+
+.four {
+  background-color: rgb(102,0,0);
+  order: 6;
+  flex-grow: 1;
+}
+
+.five {
+  background-color: rgb(51,0,0);
+  order: 5;
+  flex-grow: 1;
+}
+
+.six {
+  background-color: rgb(0,0,0);
+  order: 4;
+  flex-grow: 1;
+}
+
+.body {
+  margin: 0;
+}
```

```suggestion
body {
  margin: 0;
}
```
you can style body, html by tags, move this rule at the start of the file